style-guide/src/styles/otkit-desktop-typography.module.css

```diff
@@ -0,0 +1,10 @@
+@value spacing-large from 'otkit-spacing/token.cssmodules.css';
+
+.font-column {
+	width: 500px;
```

Comment:
fixing the spacing here

Reply:
Actually, I'm not sure, if we are going to use 8 spaces or 4 or 2, but I think generally we are using 2, so I'd like to claim we should use that. ^_^